crates/goose-cli/src/scenario_tests/scenarios.rs

```diff
@@ -8,10 +8,13 @@ mod tests {
     use crate::scenario_tests::scenario_runner::run_scenario;
     use anyhow::Result;
     use goose::conversation::message::Message;
+    use serial_test::serial;
 
     #[tokio::test]
+    #[serial]
     async fn test_what_is_your_name() -> Result<()> {
-        run_scenario(
+        std::env::set_var("GOOSE_MOIM_ENABLED", "false");
```

Comment:
as we discussed before, this is not how you want to set environment variables for a test. but also, why do we need to disable this? the point of these is to show that everything still works if you have a new feature, and then disabling the new feature while testing seems not great